hr_course/models/hr_course.py

```diff
@@ -180,6 +183,53 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_course_notification_email(self, partner_emails):
```

Comment:
```suggestion
    def send_course_notification_email(self, partner_emails):
```
partner_emails is channel now and seems we don't need send it with args

Reply:
removed param and moved handling channel inside a function as this is related strictly to way we handle notification